WalletWasabi/WabiSabi/Client/Banning/CoinPrison.cs

```diff
@@ -0,0 +1,116 @@
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using WalletWasabi.Blockchain.TransactionOutputs;
+using WalletWasabi.Helpers;
+using WalletWasabi.Logging;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.WabiSabi.Client.Banning;
+
+public class CoinPrison
+{
+	public CoinPrison(string filePath)
+	{
+		FilePath = filePath;
+	}
+
+	public List<PrisonedCoinRecord> BannedCoins { get; set; } = new();
```

Comment:
public List? Are we gonna modify this from outside?

Reply:
No, I'll change it to private.